src/main/kotlin/shared/infra/UuidV4IdentifierGenerator.kt

```diff
@@ -0,0 +1,9 @@
+package shared.infra
```

Comment:
I was thinking to delete the infra folder, since we will never test this on domain, we will test it on the android app (which will have this class, but we'll test the integration tests there, not here).

Since this pull request starts the refactoring of changing all the identifiers to Identifier (because when you want to identify something is indifferent if it is a device, a note, or myballs, you want to identify it with an identifier doesn't matter what object is being identified) .

Therefore, you can delete it on this pr and we're good to go